entries/jQuery.ajax.xml

```diff
@@ -104,6 +104,9 @@ $.ajax({
         <type name="Function"/>
         <desc>Specify the callback function name for a JSONP request.  This value will be used instead of the random name automatically generated by jQuery. It is preferable to let jQuery generate a unique name as it'll make it easier to manage the requests and provide callbacks and error handling. You may want to specify the callback when you want to enable better browser caching of GET requests. <strong>As of jQuery 1.5</strong>, you can also use a function for this setting, in which case the value of <code>jsonpCallback</code> is set to the return value of that function. </desc>
       </property>
+      <property default="'GET'" name="method" type="String" added="1.9.0">
+        <desc>The HTTP method to use for the request (e.g. "POST", "GET", "PUT"); default is "GET".</desc>
```

Comment:
I know it was just takeover from the `type` option, but some nitpicks.

>  default is "GET".

Seems like weird to say this again, as we already indicate that just a line above that: http://gyazo.com/3e298875859e35d8dfdcef35f71ab3ac

> (e.g. "POST", "GET", "PUT")

Maybe wrap these in `<code>` elements.